sdk/spring/spring-cloud-azure-starter-data-cosmos/pom.xml

```diff
@@ -32,7 +32,8 @@
   <developers>
     <developer>
       <id>microsoft</id>
```

Comment:
Could you please uniform this content in all our pom files?
For example: 
1. Delete `<id>`.
2. Put all contents together.